composer.json

```diff
@@ -52,6 +52,9 @@
 		"check-cs": [
 			"\"vendor/bin/phpcs\""
 		],
+		"check-cs-errors": [
+			"\"vendor/bin/phpcs\" --error-severity=1 --warning-severity=6"
```

Comment:
There are better ways to do this:

To show warnings, but only break the build on errors:
`--runtime-set ignore_warnings_on_exit 1`

To not show warnings at all:
`-n`